CHANGELOG.md

```diff
@@ -1,26 +1,48 @@
 # Lotus changelog
 
-# UNRELEASED
+# Node and Miner v1.30.0-rc1 / 2024-10-07
```

Comment:
I do not have any strong feelings here, I opted for having them together to lighten the workload a little. If we have separate changelog entries for each of the branches there will be some extra PRs and reviews to go through.

Reply:
I bumped both the versions in this PR - and it generates both tags even though we have targeted this PR only towards the `release/v1.30.0` branch:

- miner/v1.30.0-rc1 release tag: https://github.com/filecoin-project/lotus/releases/tag/untagged-ab3ea42d7e141069dceb
- v1.30.0-rc1 release tag: https://github.com/filecoin-project/lotus/releases/tag/untagged-dd01094414fed18ec65d

So it is only a matter of updating the `release/miner/v1.30.0` branch to be similar to the `release/v1.30.0` branch once this gets merged, and it will save us an additional round of PRs.

Reply:
Per 2024-10-08 verbal:
* 1 combined changelog entry
* 2 branches
* For network upgrades, we skip Miner version to match minimum Node version.  (I'll update docs on this.)
I'll resolve this conversation.